RiotSwiftUI/Modules/Authentication/VerifyEmail/Coordinator/AuthenticationVerifyEmailCoordinator.swift

```diff
@@ -94,6 +102,8 @@ final class AuthenticationVerifyEmailCoordinator: Coordinator, Presentable {
                 self.callback?(.cancel)
             case .goBack:
                 self.authenticationVerifyEmailViewModel.goBackToEnterEmailForm()
+            case .sendPassword(let password):
+                self.createAccount(password: password)
```

Comment:
I think we should add a comment "Tchap:..." before the case .sendPassword
I would rename the method `createAccount` with `prepareAccountCreation` because it looks like we just send the password here without actually creating an account

Reply:
These 2 points have been fixed in latests commits.